saleor/payment/gateways/braintree/__init__.py

```diff
@@ -53,7 +53,9 @@ def get_customer_data(payment_information: PaymentData) -> Dict:
             "locality": billing.city,
             "region": billing.country_area,
             "country_code_alpha2": billing.country,
-        },
+        }
+        if billing
```

Comment:
Lets create a billing structure in separate function or create billing before `return` section